docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+# Downloading Data
+
+## Downloading data from Collaboratory repository
+
+Academic research cloud infrastructure built to house ICGC data.
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#collaboratory)
+
+#### Prerequisites
+
+1.  [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data)
```

Comment:
The URL seems not correct. Do you mean this? `[Apply for DACO access](/download/data-access/#apply-for-access-to-controlled-data)`

Reply:
Corrected now